multiversx_sdk_cli/cli_wallet.py

```diff
@@ -209,6 +216,24 @@ def _create_wallet_content(
         pem = UserPEM(address.bech32(), secret_key)
         return pem.to_text()
 
+    if out_format == WALLET_FORMAT_BECH32_ADDRESS:
+        if mnemonic:
+            secret_key = mnemonic.derive_key(address_index)
+        assert secret_key is not None
+
+        pubkey = secret_key.generate_public_key()
+        address = pubkey.to_address(address_hrp)
+        return address.bech32()
+
+    if out_format == WALLET_FORMAT_PUBLIC_KEY:
+        if mnemonic:
+            secret_key = mnemonic.derive_key(address_index)
+        assert secret_key is not None
+
+        pubkey = secret_key.generate_public_key()
+        address = pubkey.to_address(address_hrp)
+        return address.hex()  # type: ignore
```

Comment:
We can return directly `pubkey.hex()`, without `address = `.

Reply:
Indeed, done!